ext/node/polyfills/fs.ts

```diff
@@ -189,6 +189,7 @@ const promises = {
   appendFile: appendFilePromise,
   readFile: readFilePromise,
   watch: watchPromise,
+  sync: fsyncSync,
```

Comment:
`promises.sync` doesn't seem existing in Node.js fs docs https://nodejs.org/api/fs.html

Or am I missing something?

Reply:
<img width="989" alt="image" src="https://github.com/user-attachments/assets/bf9a909f-4a51-48ab-b2e2-58c11556fafd">
Is this correct? I noticed that the filehandle class in your issue requires this method. Did I misunderstand? Thank you for your reply

Reply:
Oh, this is my problem. Do I need to delete it now？
Thank you for your reply

Reply:
yes

Reply:
ok, I'm done. Thank you for your reply

Reply:
Thanks for taking a look! Maybe a commit wasn't pushed correctly. This line here is still included in the changes of the PR.

Reply:
here [`be61bce` (#27089)](https://github.com/denoland/deno/pull/27089/commits/be61bcebf0c1a062a60b6c83b78a9e8c9db04752)
Is this correct ？
thank you for your replay